zebra-state/src/on_disk.rs

```diff
@@ -134,10 +151,37 @@ impl Service<Request> for SledState {
                 }
                 .boxed()
             }
+            Request::GetKnownBlockHashes { range } => {
+                let storage = self.clone();
+
+                async move {
+                    let hashes = storage.range(range)?;
+
+                    Ok(Response::KnownBlockHashes { hashes })
+                }
+                .boxed()
+            }
         }
     }
 }
 
+/// An alternate repr for `BlockHeight` that implements `AsRef<[u8]>` for usage
+/// with sled
+struct BytesHeight(u32, [u8; 4]);
+
+impl From<BlockHeight> for BytesHeight {
+    fn from(height: BlockHeight) -> Self {
+        let bytes = height.0.to_be_bytes();
```

Comment:
Why big-endian here?
Is the native byte order ok? (It might not be, if we want to allow users to move state across machines.)
Should we instead use the Zcash little-endian default?

Reply:
sled is essentially a persistent concurrent BTreeMap<[u8], [u8]>; big endian ordering means that lex ordering on byte strings is numeric ordering on u32s